CHANGELOG.md

```diff
@@ -8,7 +8,10 @@
 
 ### Packaging
 
-* MapboxCoreNavigation now depends on [MapboxNavigationNative v79._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/79.0.0). ([#3509](https://github.com/mapbox/mapbox-navigation-ios/pull/3509))
+* MapboxCoreNavigation now depends on [MapboxNavigationNative v80._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/80.0.0). ([#3590](https://github.com/mapbox/mapbox-navigation-ios/pull/3590))
```

Comment:
Consider deemphasizing the MapboxNavigationNative upgrade by putting it at the end of the list. It’s a dependency that we don’t want developers adding an explicit dependency on.

We should also be taking the time to investigate what functionally changed between the two MapboxNavigationNative versions and inlining those release notes here, since MapboxNavigationNative release notes [aren’t publicly accessible](https://github.com/mapbox/mapbox-navigation-native-ios/releases). That’s a time-consuming affair, so we should take care of it as tail work in time for the final release rather than hold up this prerelease for it.

Reply:
Done. And yes, I agree about Navigation Native changes